app/models/game.rb

```diff
@@ -1,5 +1,6 @@
 class Game < ActiveRecord::Base
-	belongs_to :user
+	belongs_to :white_player, class_name: "User"
+	belongs_to :black_player, class_name: "User"
 	has_many :pieces
```

Comment:
yes, they should complement each other.  Our database showed a game has_many :users and vice versa.  
the way this is written we can have @game.user - but who does this connect us to?  A game actually has 2 users (players) that we want to be able to connect it to.